thespian/system/transport/TCPTransport.py

```diff
@@ -340,13 +340,12 @@ def getAdminAddr(capabilities):
     @staticmethod
     def getAddressFromString(addrspec, adminRouting=False):
         if isinstance(addrspec, tuple):
-            addrparts = addrspec
+            addrparts = list(addrspec)
```

Comment:
oh, there was an intermediate version that did:
```
addrparts += [ DEFAULT_ADMIN_PORT ]
return ActorAddress(addrtype(addrparts[0], addrparts[1], external=True))
```
which required that addrparts be a list instead of (possibly) a tuple.  It works, but isn't very clear.